flink-runtime/src/test/java/org/apache/flink/runtime/state/filesystem/FsCheckpointStorageTest.java

```diff
@@ -165,8 +165,7 @@ public void testDirectoriesForExclusiveAndSharedState() throws Exception {
 		CheckpointStateOutputStream exclusiveStream =
 				storageLocation.createCheckpointStateOutputStream(CheckpointedStateScope.EXCLUSIVE);
 
-		exclusiveStream.write(42);
-		exclusiveStream.flush();
```

Comment:
If we type the `exclusiveStream` to `FsCheckpointStateOutputStream`, we could call `flushToFile()` directly.
Then we don't implicitly rely on the threshold logic here as well.

Reply:
Good point!